akka-http-core/src/main/mima-filters/10.1.11.backwards.excludes/1354-add-HttpCookie-SameSite-attribute.backwards.excludes

```diff
@@ -0,0 +1,11 @@
+# Add SameSite attribute to HttpCookie
+
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.http.javadsl.model.headers.HttpCookie.getSameSite")
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.http.javadsl.model.headers.HttpCookie.withSameSite")
+
+ProblemFilters.exclude[IncompatibleSignatureProblem]("akka.http.scaladsl.model.headers.HttpCookie.fromPair$default$8")
```

Comment:
The problem is that we cannot change these because it's a genuine binary incompatibility. We need to convert the `case class` to a regular class and keep providing those methods as deprecated until the next release cycle.